cmd/eib/main.go

```diff
@@ -54,29 +54,34 @@ func processArgs() (*image.Context, error) {
 		return nil, fmt.Errorf("building dir structure: %w", err)
 	}
 
-	return ctx, err
+	setupLogging(ctx, verbose)
+
+	return ctx, nil
+}
+
+func generateBuildLogFilename(ctx *image.Context) string {
+	const buildLogFile = "eib-build-%s.log"
+
+	timestamp := time.Now().Format("Jan02_15-04-05")
+	filename := fmt.Sprintf(buildLogFile, timestamp)
+
+	return filepath.Join(ctx.BuildDir, filename)
 }
 
-func setupLogging(verbose bool) {
+func setupLogging(ctx *image.Context, verbose bool) {
+	logFilename := generateBuildLogFilename(ctx)
+
 	logLevel := zap.InfoLevel
 	if verbose {
 		logLevel = zap.DebugLevel
 	}
 
-	encoderCfg := zap.NewProductionEncoderConfig()
-	encoderCfg.EncodeTime = zapcore.ISO8601TimeEncoder
-
-	logConfig := zap.Config{
-		Level:         zap.NewAtomicLevelAt(logLevel),
-		Encoding:      "console",
-		EncoderConfig: encoderCfg,
-		OutputPaths: []string{
-			"stdout",
-		},
-		ErrorOutputPaths: []string{
-			"stderr",
-		},
-	}
+	logConfig := zap.NewProductionConfig()
+	logConfig.Level = zap.NewAtomicLevelAt(logLevel)
```

Comment:
Should we drop the flag and use DEBUG level as a default with the introduction of user facing audits in https://github.com/suse-edge/edge-image-builder/pull/49?